src/components/dev-hub/survey-banner.js

```diff
@@ -0,0 +1,105 @@
+import React, { useEffect, useState } from 'react';
+import styled from '@emotion/styled';
+import Button from './button';
+import { fontSize, screenSize, size, layer } from './theme';
+import XWithCircleIcon from '@leafygreen-ui/icon/dist/XWithCircle';
+import IconButton from '@leafygreen-ui/icon-button';
+
+const Footer = styled('div')`
+    background-color: ${({ theme }) => theme.colorMap.greyDarkTwo};
+    position: fixed;
+    left: 0;
+    right: 0;
+    bottom: 0;
+    border: 1px solid ${({ theme }) => theme.colorMap.greyDarkOne};
+    padding: 0px ${size.medium};
```

Comment:
If we want to be meticulous we can alphabetize this CSS but given priority I am fine to leave as-is